src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private Figure[] figures = {
+            new Circle(random.nextInt(50)),
+            new IsoscelesTrapezoid(random.nextInt(50), random.nextInt(50), random.nextInt(50)),
+            new Rectangle(random.nextInt(50), random.nextInt(50)),
+            new Square(random.nextInt(50)),
+            new RightTriangle(random.nextInt(50), random.nextInt(50))};
+
+    public Figure getRandomFigure() {
+        Figure figure = figures[random.nextInt(figures.length)];
+        return figure;
+    }
+
+    public Figure getDefaultFigure() {
+        Figure circle = new Circle(10);
```

Comment:
```suggestion
        new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);
```

create constants for color and radius
Color.WHITE.name().toLowerCase()